gallery/visualtests/nxDrawer.js

```diff
@@ -134,18 +134,28 @@ describe('NxDrawer', function() {
       await checkFullPageScreenshot();
     });
 
-    it('it stays open when the toast is closed', async function() {
+    it('stays open when the toast is closed', async function() {
       await getPage().setViewport(viewportSize);
-      const [openErrorToastButton] = await waitAndGetElements('#nx-toast-error-open-button');
-      const drawer = await waitAndGetElements('#nx-drawer-with-nx-toast');
+
+      const [drawer] = await waitAndGetElements('#nx-drawer-with-nx-toast');
 
       await openDrawer(buttonId, drawerId);
+
+      const [openErrorToastButton] = await waitAndGetElements('#nx-toast-error-open-button');
+
       await openErrorToastButton.click();
 
-      const toast = await waitAndGetElements('.nx-toast-container .nx-toast');
+      const [toast] = await waitAndGetElements('.nx-toast');
+
+      await wait(500);
```

Comment:
You may not need this wait(500) at all, since the toast will render immediately, but will slowly animate to its position (I could be wrong!). If you do need it, I'd put it one line above, after openErrorToastButton.click()

Reply:
> 

I tried running it without the wait, but the the expect for if the toast isVisible fails. I moved that line up here: 95752e9a5cf87abd4d87c552048814cf10dc28f3